scripts/checkpatch.sh

```diff
@@ -2,6 +2,10 @@
 
 DIR="${BASH_SOURCE%/*}"
 
+if [ -z "$CHECKPATCH" ]; then
+  CHECKPATCH=$(find /usr/src/linux-headers* -name checkpatch.pl -print -quit)
+fi
```

Comment:
With this, `/usr/src/...` will have higher priority than `$PATH`. Not convenient I think.
How about just adding`PATH=$PATH:/usr/src/linux-headers*/scripts` in `checkpatch_inc.sh` instead?

Reply:
Wildcards in `PATH` have their own issues. And while I don't exoect any weird sideeffect, I prefer to not clutter the `PATH` variable.
Updated version of PR respects any existing additions to path and adds another fallback.

Reply:
Sounds reasonable to me. Would you please squash the patches into one and write a proper commit message?

Reply:
Will do. I'll have to come up with a commit message that soenst mention checkpatch, because checkpatch doesn't like being mentioned in commit message subjects. (This is why the CI builds fail atm,)

Reply:
@jforissier Done.